src/main/java/com/dzionek/simpleshell/commands/affecting/AffectingCommandsFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.dzionek.simpleshell.commands.affecting;
+
+import com.dzionek.simpleshell.commands.Commands;
+import com.dzionek.simpleshell.commands.Factory;
+import com.dzionek.simpleshell.shellmanaging.ShellData;
+
+/**
+ * @author Bartlomiej Janik
+ * @since 9/16/2017
+ */
+public class AffectingCommandsFactory implements Factory<SetterCommand> {
+    @Override
+    public SetterCommand pickProperCommand(ShellData shellData, Commands command) {
+        return shellData.getMapOfAffectingCommands().get(command);
```

Comment:
prawo demeter 
-> w efektywnej javie do sprawdzenia czy to juz podchodzi pod prawo demeter
-> https://pl.wikipedia.org/wiki/Prawo_Demeter
-> https://alvinalexander.com/java/java-law-of-demeter-java-examples  (get().get(), look at "The Law of Demeter - Bad examples")